frontend/apps/filemanager/filemanagerhistory.lua

```diff
@@ -51,16 +77,16 @@ function FileManagerHistory:updateItemTable()
         reading = 0, abandoned = 0, complete = 0, deleted = 0, new = 0, }
     local item_table = {}
     for _, v in ipairs(require("readhistory").hist) do
-        if not self.filter or v.status == self.filter then
+        if self.filter == "all" or v.status == self.filter then
             table.insert(item_table, v)
         end
         if self.statuses_fetched then
             self.count[v.status] = self.count[v.status] + 1
         end
     end
     local title = self.hist_menu_title
-    if self.filter then
-        title = title .. " (" .. status_text[self.filter] .. ")"
+    if self.filter ~= "all" then
+        title = title .. " (" .. filter_text[self.filter] .. ": " .. self.count[self.filter] .. ")"
```

Comment:
I was going to say this might need some thinking/wrapping for BiDi - but trying it with Hebrew & Arabic, it seems to work as is and look as expected in RTL:
![image](https://user-images.githubusercontent.com/24273478/205328616-a116ce49-340c-4605-91dd-9b2b0b560d71.png)